gen3/lib/fence/config-helper.py

```diff
@@ -69,7 +71,7 @@ def _nested_extract_config(template, source):
     result_content = {}
     try:
         for inner_key, inner_value in template.items():
-            if source is not None and inner_key in source and source[inner_key] is not None:
+            if source.get(inner_key) is not None:
```

Comment:
sorry, i was wrong in my previous comment, i didn't see you also want to check `source is not None`...
`source.get(inner_key)` does not check for that so you should add it back if it's needed